### PR TITLE
Improve wording for the new frame and equipotentials and translate

### PR DESCRIPTION
**Pull request by eggrobin**
#3358.

### CODE REVIEW

File: ksp_plugin_adapter/localization/zh-cn.cfg [hunk 1]
```diff
@@ -73,16 +77,22 @@ Localization {
...
     #Principia_ReferenceFrameSelector_Description_BodyCentredParentDirection = <<1>>的质心已固定，\n水平面为绕着<<3>>的轨道。\n这个参考系跟随轨道旋转，面向着<<3>>的方位不变。
     #Principia_ReferenceFrameSelector_Description_BodySurface = <<1>>已固定，\n水平面为它的赤道。\n这个参考系与<<1>>共同旋转。
+    // TODO(Al₂Me₆): Translate these.
+    #Principia_ReferenceFrameSelector_Description_RotatingPulsating = The centres of <<A:3>> and <<A:1>> are fixed.\nThe plane of their orbit is horizontal.\nThe reference frame rotates with the orbit and pulsates to keep the <<nom:3>>–<<nom:1>> distance constant.
+    #Principia_ReferenceFrameSelector_Description_RotatingPulsatingOneMoon = The centre of <<A:3>> and the barycentre of the <<nom:1>>–<<nom:5>> system are fixed.\nThe plane of their orbit is horizontal.\nThe reference frame rotates with the orbit and pulsates to keep the <<nom:3>>–<<nom:1>> distance constant.
+    #Principia_ReferenceFrameSelector_Description_RotatingPulsatingBigSystem = The centre of <<A:3>> and the barycentre of the <<nom:1>> system are fixed.\nThe plane of their orbit is horizontal.\nThe reference frame rotates with the orbit and pulsates to keep the <<nom:3>>–<<nom:1>> distance constant.
```
Comment (al2me6) [inline ksp_plugin_adapter/localization/zh-cn.cfg:83]:
```suggestion
    #Principia_ReferenceFrameSelector_Description_RotatingPulsating = <<3>>与<<1>>的质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道同步旋转并且脉动，以使<<3>>-<<1>>距离恒定。
    #Principia_ReferenceFrameSelector_Description_RotatingPulsatingOneMoon = <<3>>的质心与<<1>>-<<5>>系统质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道同步旋转并且脉动，以使<<3>>-<<1>>距离恒定。
    #Principia_ReferenceFrameSelector_Description_RotatingPulsatingBigSystem = <<3>>的质心与<<1>>系统质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道同步旋转并且脉动，以使<<3>>-<<1>>距离恒定。
```

(Yes, both 已 and 以 are intentional this time XD)

File: ksp_plugin_adapter/localization/zh-cn.cfg [hunk 2]
```diff
@@ -73,16 +77,22 @@ Localization {
...
     #Principia_ReferenceFrameSelector_ReferencePlane = <<1>>绕<<2>>的轨道  // <<1>> secondary body <<2>> centred body
     #Principia_ReferenceFrameSelector_ToggleButton = <<1>>选择器（<<2>>）...  // <<1>> plotting <<2>> selected frame
     #Principia_ReferenceFrameSelector_Title = <<1>>选择器（<<2>>）  // <<1>>plotting <<2>> selected frame
-    // TODO(egg): translations.
+
     #Principia_ReferenceFrameSelector_Tooltip_BodyCentredNonRotating = <<1>>参考系：<<2>>为中心，轨迹画成有摄动的Kepler轨道。
     #Principia_ReferenceFrameSelector_Tooltip_BodySurface = <<1>>参考系：用于有关<<2>>表面的活动：着陆、遥感、观测、通讯等等。
     #Principia_ReferenceFrameSelector_Tooltip_BodyCentredParentDirection = <<1>>参考系：用于飞略<<2>>和<<2>>为起讫点的轨道转移。
+    // TODO(Al₂Me₆): Translate.
+    #Principia_ReferenceFrameSelector_Tooltip_RotatingPulsating = <<1>> reference frame: for low-energy transfers or orbits about the <<nom:3>>–<<nom:2>> Lagrange points.
```
Comment (al2me6) [inline ksp_plugin_adapter/localization/zh-cn.cfg:95]:
```suggestion
    #Principia_ReferenceFrameSelector_Tooltip_RotatingPulsating = <<1>>参考系：用于低能量转移轨道或绕于<<3>>-<<2>>拉格朗日点的轨道。
```

File: ksp_plugin_adapter/localization/en-us.cfg [hunk 3]
```diff
@@ -75,7 +75,9 @@ Localization {
     #Principia_ReferenceFrameSelector_Description_BodyCentredNonRotating = The centre of <<A:1>> is fixed.\nIts equator is horizontal.\nThe reference frame does not rotate.
     #Principia_ReferenceFrameSelector_Description_BodyCentredParentDirection = The centre of <<A:1>> is fixed.\nThe plane of its orbit around <<A:3>> is horizontal.\nThe reference frame rotates with the orbit, fixing the direction of <<A:3>>.
     #Principia_ReferenceFrameSelector_Description_BodySurface = <<AC:1>> is fixed.\nIts equator is horizontal.\nThe reference frame rotates with <<A:1>>.
-    #Principia_ReferenceFrameSelector_Description_RotatingPulsating = The centres of <<A:3>> and <<A:1>> are fixed.\nThe plane of their orbit is horizontal.\nThe reference frame rotates with the orbit and pulsates, fixing the centres of both bodies.
+    #Principia_ReferenceFrameSelector_Description_RotatingPulsating = The centres of <<A:3>> and <<A:1>> are fixed.\nThe plane of their orbit is horizontal.\nThe reference frame rotates with the orbit and pulsates to keep the <<nom:3>>–<<nom:1>> distance constant.
```
Comment (al2me6) [inline ksp_plugin_adapter/localization/en-us.cfg:78]:
```suggestion
    #Principia_ReferenceFrameSelector_Description_RotatingPulsating = The centres of <<A:3>> and <<A:1>> are fixed.\nThe plane of their orbit is horizontal.\nThe reference frame rotates and pulsates with the orbit of <<A:3>> such that the direction of <<A:3>> and the <<nom:3>>–<<nom:1>> distance are both fixed.
```
Perhaps its similarity to <secondary>-<primary> orbit should be emphasized?

File: ksp_plugin_adapter/main_window.cs [hunk 4]
```diff
@@ -219,23 +219,23 @@ internal class MainWindow : VesselSupervisedWindowRenderer {
           if (UnityEngine.GUILayout.Button(
                   L10N.CacheFormat("#Principia_MainWindow_Declutter"),
                   GUILayoutWidth(5))) {
-            show_only_pinned_markers = true;
-            show_only_pinned_celestials = true;
-            hide_equipotentials = true;
+            show_unpinned_markers = false;
+            show_unpinned_celestials = false;
+            show_equipotentials = false;
```
Comment (pleroy) [inline ksp_plugin_adapter/main_window.cs:224]: I am noticing that this state is not persisted.  It should be, it's not nice for the user to have to unclutter each time.  Please update `Save` and `Load` accordingly.

Comment (eggrobin) [reply ksp_plugin_adapter/main_window.cs:224]: It should not be; the point of decluttering is that it is highly situational, and not very fine grained.
We don’t want people who pressed the « get the noodles out of my face » button once when they were looking at a long flight plan, to never see a single equipotential.

Comment (pleroy) [reply ksp_plugin_adapter/main_window.cs:224]: Ack.

**Review by eggrobin — COMMENTED** [review]
Suggestion from @Test-Account-KSP.

File: ksp_plugin_adapter/localization/en-us.cfg (hunk 3 above)
Comment (eggrobin) [reply ksp_plugin_adapter/localization/en-us.cfg:78]: This is tricky.
Normally the descriptions are:
1. Fixed point
2. Reference plane
3. Reference direction within that plane.

However, by giving two fixed points, we have already determined everything but the reference frame; saying that it fixes the distance is already redundant (but pulsating is weird, so I called it out). The reference direction is even more redundant.

Comment (al2me6) [reply ksp_plugin_adapter/localization/en-us.cfg:78]: Agreed. I'm just not sure if the implications are entirely obvious to the average user.

File: ksp_plugin_adapter/localization/celestial_strings.cfg [hunk 5]
```diff
@@ -128,7 +128,15 @@
...
     #Principia_ReferenceFrameSelector_Name_BodySurface(Moon).zh-cn = 月心月固
     #Principia_ReferenceFrameSelector_Name_BodyCentredParentDirection(Earth,Sun).zh-cn = 地心太阳黄道
+    #Principia_ReferenceFrameSelector_Name_RotatingPulsating(Moon,Earth).zh-cn = 地月拉格朗日点
+    #Principia_ReferenceFrameSelector_Name_RotatingPulsating(Earth,Sun).zh-cn = 日地拉格朗日点
     #Principia_ReferenceFrameSelector_NavballName_BodyCentredParentDirection(Earth,Sun).zh-cn = 地心太阳黄道
+    #Principia_ReferenceFrameSelector_NavballName_RotatingPulsating(Moon,Earth).zh-cn = 地月拉格朗日点
+    #Principia_ReferenceFrameSelector_NavballName_RotatingPulsating(Earth,Sun).zh-cn = 日地拉格朗日点
     #Principia_ReferenceFrameSelector_Description_BodyCentredParentDirection(Earth,Sun).zh-cn = <<1>>的质心已固定，\n水平面为黄道（<<1>>绕<<3>>的轨道面）。\n这个参考系跟随轨道旋转，面向着<<3>>的方位不变。
+    #Principia_ReferenceFrameSelector_Description_RotatingPulsating(Moon,Earth).zh-cn = <<3>>的质心与<<1>>的质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道同步旋转并且脉动，以使地月距离恒定。
+    #Principia_ReferenceFrameSelector_Description_RotatingPulsatingOneMoon(Earth,Sun,Moon).zh-cn = <<3>>的质心与地月系质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道同步旋转并且脉动，以使日地距离恒定。
```
Comment (eggrobin) [inline ksp_plugin_adapter/localization/celestial_strings.cfg:138]: @al2me6 These need to be updated, probably to say something about the 日地线 and 地月线.

File: ksp_plugin_adapter/localization/zh-cn.cfg [hunk 6]
```diff
@@ -73,16 +77,20 @@ Localization {
     #Principia_ReferenceFrameSelector_Description_BodyCentredNonRotating = <<1>>的质心已固定，\n水平面为它的赤道。\n这个参考系不旋转。
     #Principia_ReferenceFrameSelector_Description_BodyCentredParentDirection = <<1>>的质心已固定，\n水平面为绕着<<3>>的轨道。\n这个参考系跟随轨道旋转，面向着<<3>>的方位不变。
     #Principia_ReferenceFrameSelector_Description_BodySurface = <<1>>已固定，\n水平面为它的赤道。\n这个参考系与<<1>>共同旋转。
+    #Principia_ReferenceFrameSelector_Description_RotatingPulsating = <<3>>的质心与<<1>>的质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道同步旋转并且脉动，以使<<3>>-<<1>>距离恒定。
+    #Principia_ReferenceFrameSelector_Description_RotatingPulsatingOneMoon = <<3>>的质心与<<1>>-<<5>>系统质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道同步旋转并且脉动，以使<<3>>-<<1>>距离恒定。
+    #Principia_ReferenceFrameSelector_Description_RotatingPulsatingBigSystem = <<3>>的质心与<<1>>系统质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道同步旋转并且脉动，以使<<3>>-<<1>>距离恒定。
```
Comment (eggrobin) [inline ksp_plugin_adapter/localization/zh-cn.cfg:82]: @al2me6 These need to be updated to say something about the line between <<3>> and <<1>>.

File: ksp_plugin_adapter/localization/celestial_strings.cfg (hunk 5 above)
Comment (al2me6) [reply ksp_plugin_adapter/localization/celestial_strings.cfg:138]:
```suggestion
    #Principia_ReferenceFrameSelector_Description_RotatingPulsating(Moon,Earth).zh-cn = <<3>>的质心与<<1>>的质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道旋转，以使地月线固定，并且脉动，以使地月距离恒定。
    #Principia_ReferenceFrameSelector_Description_RotatingPulsatingOneMoon(Earth,Sun,Moon).zh-cn = <<3>>的质心与地月系质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道旋转，以使日地线固定，并且脉动，以使日地距离恒定。
```

File: ksp_plugin_adapter/localization/zh-cn.cfg (hunk 6 above)
Comment (al2me6) [reply ksp_plugin_adapter/localization/zh-cn.cfg:82]:
```suggestion
    #Principia_ReferenceFrameSelector_Description_RotatingPulsating = <<3>>的质心与<<1>>的质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道旋转，以使<<3>>-<<1>>连线固定，并且脉动，以使<<3>>-<<1>>距离恒定。
    #Principia_ReferenceFrameSelector_Description_RotatingPulsatingOneMoon = <<3>>的质心与<<1>>-<<5>>系统质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道旋转，以使<<3>>-<<1>>连线固定，并且脉动，以使<<3>>-<<1>>距离恒定。
    #Principia_ReferenceFrameSelector_Description_RotatingPulsatingBigSystem = <<3>>的质心与<<1>>系统质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道旋转，以使<<3>>-<<1>>连线固定，并且脉动，以使<<3>>-<<1>>距离恒定。
```

**Pull request merged**